src/extensions/replay/sessionrecording.ts

```diff
@@ -584,6 +616,16 @@ export class SessionRecording {
         })
     }
 
+    private _scheduleFullSnapshot(): void {
+        if (this._fullSnapshotTimer) {
+            clearInterval(this._fullSnapshotTimer)
+        }
+
+        this._fullSnapshotTimer = setInterval(() => {
+            this._tryTakeFullSnapshot()
+        }, 1000 * 60 * 5) // 5 minutes
```

Comment:
Move this to a constant, just a bit though, you spare the calculation on every method call unless the compiler already optimizes that.

Reply:
I'd hope the compiler figures it out... but I already felt the need to add a comment... so a constant is less effort without having to guess what the compiler does 😊

Reply:
and I guess multiple browsers means multiple possible compilers so multiple behaviours